extension/observer/dockerobserver/config_test.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/extension/observer/dockerobserver/internal/metadata"
 )
 
-var version = "1.40"
+var version = "1.44"
```

Comment:
Is there a reason we're updating this version? I believe we want to keep `1.40` for the sake of testing, to make sure it's interpreted properly (as noted in the README). I may be missing something though.

Reply:
This does not hurt but I agree with you I guess no reason to complicate stuff.